src/webapi/main.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+
+class Bot(BaseModel):
+    name: str
+
+
+app = FastAPI(
+    title='Discord Bot Database',
+    docs_url='/'
+)
+
+
+@app.get('/api/bots')
+def get_bots():
+    return {'message': 'Hello World'}
+
+
+@app.get('/api/bots/{id}')
+def get_bot(id: int):
+    return {'message': 'Hello World'}
+
+
+@app.post('/api/bots')
```

Comment:
```suggestion
@app.post('/api/bots', status_code=201)
```
post して作られたら 201 返したほうが良さそう？(ヘッダでなんか与えてあげないといけなさそう)

Reply:
> **201 Created**
>     **作成。**リクエストは完了し、新たに作成されたリソースのURIが返される。
>     例: PUTメソッドでリソースを作成するリクエストを行ったとき、そのリクエストが完了した場合に返される。

よさそうですね！

Reply:
POSTとPUTは200/201/204で少し迷いましたが、
今回のケースだとPOSTが新規レコード発行、PUTが更新を担う形になるので、
それぞれ201と200にしました。